app/src/main/java/net/gsantner/markor/frontend/textview/HighlightingEditor.java

```diff
@@ -297,6 +299,60 @@ private int rowEnd(final int y) {
         return layout.getLineEnd(line);
     }
 
+    // Text-Casing
+    // ---------------------------------------------------------------------------------------------
+    public void toggleCase() {
```

Comment:
All of this functionality should be in TextViewUtils IMO. 

A good bit of it already exists in TextViewUtils